src/components/Table/_pagination.scss

```diff
@@ -0,0 +1,21 @@
+@import '~carbon-components/scss/globals/scss/vars';
+@import '../../globals/vars';
+
+.#{$iot-prefix}--pagination {
+  .bx--pagination__left {
+    margin: auto auto auto 0;
+  }
+  .bx--select .bx--select-input ~ .bx--select__arrow {
+    align-self: center;
+  }
+  .bx--pagination__left,
+  .bx--pagination__text {
+    display: var(--pagination-text-display);
```

Comment:
Where is this variable declared/set? I can't find a reference to it

Reply:
i believe this is set in `Pagination.jsx`.

Reply:
Ah, yes nevermind